app/src/routes/$id.inntekt-og-refusjon.tsx

```diff
@@ -40,7 +43,7 @@ export type InntektOgRefusjonForm = {
 } & Pick<InntektsmeldingSkjemaState, "naturalytelserSomMistes">;
 
 function InntektOgRefusjon() {
-  const opplysninger = Route.useLoaderData();
+  const opplysninger = useLoaderData({ from: "/$id" });
```

Comment:
Kanskje vi skulle dratt ut dette som en `useOpplysninger` hook eller så? Så slipper nestemann i kodebasen kognitivt å forstå at opplysninger kommer fra $id.tsx loaderen?

Sparer jo ikke noen kodelinjer, men jeg ser for meg at det kanskje blir mer lesbar kode av det?

Reply:
Både og. Da skjuler man at du må være på en subrute av $id for å bruke den hooken